mftf/2.3/reporting.md

```diff
@@ -0,0 +1,367 @@
+ Test  tests/functional/Magento/FunctionalTest/_generated/default/AdminMenuNavigationWithSecretKeysTestCest.php:AdminMenuNavigationWithSecretKeysTest
+ Step  See "#something"
+ Fail  Failed asserting that  on page /admin/admin/system_config/index/key/678b7ba922c.../
+--> DASHBOARD
+SALES
+CATALOG
+CUSTOMERS
+MARKETING
+CONTENT
+REPORTS
+STORES
+SYSTEM
+FIND PARTNERS & EXTENSIONS
+Configuration
+admin
+1
+Store View: Default Config
+What is this?
+Save Config
+Country Options
+State Options
+Locale Options
+Store Information
+Store Name
+Store Phone Number
+Store Hours of Operation
+Countr
+[Content too long to display. See complete response in '/Users/dmytroshevtsov/Projects/vagrant/vagrant-magento/magento2ce/dev/tests/acceptance/tests/_output/' directory]
+--> contains "#something".
+
+Scenario Steps:
+
+ 23. $I->amOnPage("/admin/admin/auth/logout/") at tests/functional/Magento/FunctionalTest/_generated/default/AdminMenuNavigationWithSecretKeysTestCest.php:54
+ 22. // Cleaned cache types:
+config
+full_page
+ 21. $I->magentoCLI("cache:clean config full_page") at tests/functional/Magento/FunctionalTest/_generated/default/AdminMenuNavigationWithSecretKeysTestCest.php:52
+ 20. // Value was saved.
+ 19. $I->magentoCLI("config:set admin/security/use_form_key 0") at tests/functional/Magento/FunctionalTest/_generated/default/AdminMenuNavigationWithSecretKeysTestCest.php:50
+ 18. $I->saveScreenshot() at tests/functional/Magento/FunctionalTest/_generated/default/AdminMenuNavigationWithSecretKeysTestCest.php:63
+```
+{: .no-copy }
+
+- `1) AdminMenuNavigationWithSecretKeysTestCest: Admin menu navigation with secret keys test` - the failed Codeception test is *AdminMenuNavigationWithSecretKeysTestCest*. It references to the PHP class that implemented the failed test.
+
+- `Test  tests/functional/Magento/FunctionalTest/_generated/default/AdminMenuNavigationWithSecretKeysTestCest.php:AdminMenuNavigationWithSecretKeysTest` - the test is implemented in the *AdminMenuNavigationWithSecretKeysTest* test method of the *tests/functional/Magento/FunctionalTest/_generated/default/AdminMenuNavigationWithSecretKeysTestCest.php* file under `<magento root>/dev/tests/acceptance/`.
+  It matches the corresponding test defined in XML that is *AdminMenuNavigationWithSecretKeysTest* defined in `<test name="AdminMenuNavigationWithSecretKeysTest">...</test>`
+
+- `Step  See "#something"` - the failing test step is the *see* action with the *#something* selector. It would correspond the `<see selector="#something" ... />` test step in the XML defined tests.
+
+- `Fail  Failed asserting that  on page /admin/admin/system_config/index/key/678b7ba922c.../` - the fail occurred on the web page `<MAGENTO_BASE_URL>/admin/admin/system_config/index/key/678b7ba922c.../`.
+
+```terminal
+--> ...
+[Content too long to display. See complete response in '/Users/dmytroshevtsov/Projects/vagrant/vagrant-magento/magento2ce/dev/tests/acceptance/tests/_output/' directory]
```

Comment:
Replace most of the path with `<magento root>`

Reply:
I want to keep this report as close as possible to the real one. Will change to '/../../magento2/dev/tests/acceptance/tests/_output/'